spyder/config/main.py

```diff
@@ -93,6 +93,8 @@
               'cpu_usage/timeout': 2000,
               'use_custom_margin': True,
               'custom_margin': 0,
+              'use_custom_caret_blinking': False,
+              'custom_caret_blinking': 1000,

```

Comment:
Instead of using `caret` as part of these options, please use `cursor` instead, i.e. `use_custom_cursor_blinking` and `custom_cursor_blinking`.